acsm_scrape_petsc_configure.m4

```diff
@@ -113,18 +113,18 @@ AC_DEFUN([ACSM_SCRAPE_PETSC_CONFIGURE],
           dnl Set some include and link variables by building and running temporary Makefiles.
```

Comment:
Let's put a comment here about the SHELL= setting?  It's a really weird thing to have to do and it wouldn't hurt to explain why we actually have to do it.

Reply:
Okie doke, comment added